### DISCUSSION TITLE
No way to change URL for redirect after changing password

### DISCUSSION THREAD

**Discussion by kyleshepherd** [laravel/nova-issues#3268] [Q&A] [answered]
- Laravel Version: 8.24.0
- Nova Version: 3.10.0
- PHP Version: 7.4.

### Description:
I want to change the page a user is redirected after successfully changing their password, but I can't find a way to do this on the docs or through my own investigation. Is this currently possible with Laravel Nova? If not is there a hacky way to get this working?

**Comment by dgoldstein89** [comment, marked as answer]
Create your own ResetPasswordController which extends \Laravel\Nova\Http\Controllers\ResetPasswordController and override redirectPath(). You also have to override boot() in NovaServiceProvider and add an alias.

app/Http/Controllers/ResetPasswordController.php
```
<?php

namespace App\Http\Controllers;

class ResetPasswordController extends \Laravel\Nova\Http\Controllers\ResetPasswordController
{
    public function redirectPath()
    {
        return '/my/custom/path';
    }
}
```

app/Providers/NovaServiceProvider
```
<?php

namespace App\Providers;

class NovaServiceProvider extends NovaApplicationServiceProvider
{
    public function boot()
    {
        parent::boot();

        $this->app->alias(
            \App\Http\Controllers\ResetPasswordController::class,
            \Laravel\Nova\Http\Controllers\ResetPasswordController::class
        );
    }
}

```

**Reply by kyleshepherd** [reply]
Ah thank you, I was trying to do something like this but didn't set the `alias`. I'll attempt to implement this now and let you know if it works for me

**Reply by kyleshepherd** [reply]
This worked a treat, thanks!

**Reply by dgoldstein89** [reply]
Glad to help
